pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -86,6 +86,11 @@
     private int backlogQuotaCheckIntervalInSeconds = 60;
     // Default per-topic backlog quota limit
     private long backlogQuotaDefaultLimitGB = 50;
+    //Default backlog quota retention policy. Default is producer_request_hold
+    //'producer_request_hold' Policy which holds producer's send request until the resource becomes available (or holding times out)
+    //'producer_exception' Policy which throws javax.jms.ResourceAllocationException to the producer
+    //'consumer_backlog_eviction' Policy which evicts the oldest message from the slowest consumer's backlog
+    private String backlogQuotaRetentionPolicy = "producer_request_hold";
```

Comment:
This could be set with the enum so that we can validate the value when loading the config file.

Also, similar to the `backlogQuotaDefaultLimitGB` this config var could be called `backlogQuotaDefaultRetentionPolicy`

Reply:
Yeah, it's already have a enum in BacklogQuota named RetentionPolicy
```java
public static enum RetentionPolicy {
        /** Policy which holds producer's send request until the resource becomes available (or holding times out) */
        producer_request_hold,

        /** Policy which throws javax.jms.ResourceAllocationException to the producer */
        producer_exception,

        /** Policy which evicts the oldest message from the slowest consumer's backlog */
        consumer_backlog_eviction,
    }
```